examples/fhir/lab_order.json

```diff
@@ -11,6 +11,15 @@
       "resource": {
         "resourceType": "MessageHeader",
         "id": "665fe0e7-5681-48cc-9953-366c0712650e",
+        "meta": {
+          "tag": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v2-0103",
+              "code": "P",
```

Comment:
Does the ReportStream system use the other possible codes in this Code System?  Like `N` for non-production testing?